tk3_no1.c

```diff
@@ -0,0 +1,67 @@
+/*
+
+Pada sebuah super market membuat program hadiah dengan memberikan kupon undian kepada pembeli.
+Kupon undian akan diberikan jika total pembelian minimal Rp. 100.000,00 dan dan akan dihitung kelipatannya. Diskon 5% jika tota pembelian minimal Rp. 100.000,00.
+Misal : seorang pembeli dengan total pembelian Rp. 250.000,00 maka dapat kupon undian sebanyak 2 lembar. Dan dapat diskon sebesar Rp. 25.000,00,
+maka yang dibayar sebesar Rp. 225.000,00.
+
+Revisi: Diskon fix 10%
+
+*/
+
+/*
+  I created the program based on revision during class with Pak Tri
+*/
+
+#include <stdio.h>
+
+const float discount = 0.10;
+
+/* This is a function to determine how many coupons the user will get */
```

Comment:
I think this function is pretty simple and straightforward. Since the function name already describes what this function does, I guess the comment is redundant and unnecessary.

_Clean Code by Robert C. Martin is a good code for reference to write a clean code: https://thixalongmy.haugiang.gov.vn/media/1175/clean_code.pdf_